packages/react/src/components/PageHeader/PageHeader.stories.js

```diff
@@ -381,7 +457,15 @@ export const TabBar = (args) => {
 
 export const DirectExports = (args) => (
   <PageHeaderDirect {...args}>
-    <PageHeaderBreadcrumbBar />
+    <PageHeaderBreadcrumbBar
+      renderIcon={BreadcrumbBeeIcon}
+      contextualActions={breadcrumbContextualActions}
```

Comment:
```suggestion
      contextualActions={breadcrumbContextualActions}
```
<img width="1663" alt="image" src="https://github.com/user-attachments/assets/4f258001-325a-4a83-89e6-12b378fbcc91" />
Hey this looks great - just direct-exports storybook example is breaking (breadcrumbContextualActions not defined)

Reply:
whoops! I decided to change the name to `breadcrumbContentActions` for when we eventually get the collapsible behavior working - the actions will be coming from the content section. Forgot to swap it out there too, thanks for catching that!